test/framework/src/main/java/org/elasticsearch/indices/analysis/AnalysisFactoryTestCase.java

```diff
@@ -191,6 +191,8 @@ private static String toCamelCase(String s) {
         .put("flattengraph",              MovedToAnalysisCommon.class)
 
         // TODO: these tokenfilters are not yet exposed: useful?
+        .put("bengalinormalization",      Void.class)
+        .put("bengalistem",               Void.class)
```

Comment:
As these above tokenfilters are not exposed, You can add it later. And as this commit is related with version upgrade and this change is not compilation broker, you may consider it in future. Also, in few commits like https://github.com/elastic/elasticsearch/pull/26527  (Adding Bengali Analyzer) this may cause conflicts.

Reply:
Thanks for pointing this out. I think it is a good to add these token filters after this and your PR have been merged.